liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojo.java

```diff
@@ -224,6 +200,8 @@ private void installLooseApplication(MavenProject proj, File looseConfigFile) th
                 deleteApplication(new File(serverDirectory, "apps"), looseConfigFile);
                 deleteApplication(new File(serverDirectory, "dropins"), looseConfigFile);
                 config.toXmlFile(looseConfigFile);
+                //Only checks if server is running
+                verifyAppStarted(application);
```

Comment:
Just an observation...the only thing different in the war and ear block of this switch statement is that one calls installLooseConfigWar and the other calls installLooseConfigEar. You could make a method for the 7 lines of code and just have one conditional statement for the different install method.

Reply:
I feel like that might make the logic a bit convoluted as we'd still be handling the other two cases here. Much easier to read and follow this way.

Reply:
Ended up pulling the last 4 lines out into a method as they don't depend on packaging type.